projects/plugins/jetpack/tools/webpack.config.extensions.js

```diff
@@ -212,21 +212,13 @@ module.exports = [
 								return metadata;
 							}
 
-							let scriptName = 'editor';
-
-							if ( presetIndex.beta.includes( name ) ) {
-								scriptName += '-beta';
-							} else if ( presetIndex.experimental.includes( name ) ) {
-								scriptName += '-experimental';
-							}
-
 							// `editorScript` is required for block.json to be valid and WordPress.org to be able
 							// to parse it before building the page at https://wordpress.org/plugins/jetpack/.
 							// Don't add other scripts or styles while block assets are still enqueued manually
 							// in the backend.
 							const result = {
 								...metadata,
-								editorScript: `file:../${ scriptName }.js`,
+								editorScript: `jetpack-blocks-editor`,
```

Comment:
Prevent multiple asset downloads for now since the script is loaded manually on the backend. An `editorScript` key is required for the `block.json` file to be valid and for WordPress.org to be able to parse it.